docs/concepts/services-networking/README.md

```diff
@@ -1,3 +1,26 @@
-# Services & Networking
+# Services, Load Balancing, and Networking
 
-// TODO ([github.com/vmware-tanzu/vm-operator#111](https://github.com/vmware-tanzu/vm-operator/issues/111))
+## The VM Operator network model
+
+By default, every virtual machine (VM) in a namespace receives its own, unique, namespace-wide IP address. It is possible to deploy VM workloads that have no networking capabilities whatsoever, but this would be an explicit choice.
```

Comment:
Most commonly will but the IP isn't necessarily unique within a namespace since the namespace can have multiple networks.

Or maybe I'm misreading if "By default" means the default network that each provider selects - then n/m this comment.